h2o-docs/src/booklets/v2_2015/source/GLMBooklet.tex

```diff
@@ -571,6 +571,100 @@ \subsubsection{Tweedie Models}
 \centering
 \includegraphics[scale=0.5]{../../../product/images/tweedie_deviance.png}
 \end{figure}
+
+
+\paragraph{Tweedie Dispersion Parameter Estimation}
+
+The density for the maximum likelihood function for Tweedie can be written as:
+
+$$f( y; \theta, \phi) = a (y, \phi, p) \exp \Big[ \frac{1}{\phi} \big\{ y \theta - k(\theta) \big\} \Big] \quad \text{Equation 1}$$
+
+where:
+
+\begin{itemize}
+\item $a (y, \phi, p), k(\theta)$ are suitable known functions
+\item $\phi$ is the dispersion parameter and is positive
+\item $\theta = \begin{cases} \frac{\mu ^{1-p}}{1-p} & p \neq 1 \\ \log (\mu) & p = 1 \\\end{cases}$
+\item $k(\theta) = \begin{cases} \frac{\mu ^{2-p}}{2-p} & p \neq 2 \\ \log (\mu) & p=2 \\\end{cases}$
+\item the value of $\alpha (y,\phi)$ depends on the value of $p$
+\end{itemize}
+
+If there are weights introduced to each data row, $equation 1$ will become:
+
+$$f \Big( y; \theta, \frac{\phi}{w} \Big) = a \Big( y, \frac{\phi}{w}, p \Big) \exp \Big[ \frac{w}{\phi} \big\{ y\theta - k(\theta) \big\} \Big]$$
+
+\subparagraph{$\alpha (y, \phi) when 1<p<2$}
+
+For $Y=0$,
+
+$$P(Y=0) = \exp \Big\{-\frac{\mu^{2-p}}{\phi (2-p)} \Big\} \quad \text{Equation 2}$$
+
+For $Y>0$,
+
+$$a(y, \phi, p) = \frac{1}{y} W(y, \phi, p) \quad \text{Equation 3}$$
+
+with $W(y, \phi, p) = \sum^{\infty}_{j=1} W_j$ and 
+
+$$W_j = \frac{y^{-j \alpha}(p-1)^{\alpha j}}{\phi^{j(1-\alpha)} (2-p)^j j!T(-j\alpha)} \quad \text{Equation 4}$$
+
+If weight is applied to each row, $equation 4$ becomes:
+
+$$W_j = \frac{w^{j(1-\alpha)}y^{-j \alpha}(p-1)^{\alpha j}}{\phi^{j(1-\alpha)}(2-p)^j j!T(-j \alpha)} \quad \text{Equation 5}$$
+
+The $W_j$ terms are all positive. The following figure plots for $\mu = 0.5, p=1.5, \phi =1. y=0.1$.
+
+\begin{figure}[h]
+\centering
+\includegraphics[scale=0.25]{images/dispersion_param_fig1.png}
+\end{figure}
+
+\subparagraph{$\alpha (y, \phi) \text{ when } p > 2$}
+
+Here, you have,
+
+$$a(y, \phi, p) = \frac{1}{\pi y}V(y,\phi, p) \quad \text{Equation 6}$$
+
+and $V = \sum^{\infty}_{k=1} V_k$ where
+
+$$V_k = \frac{T(1+\alpha k)\phi^{k(\alpha - 1)}(p-1)^{\alpha k}}{T(1+k)(p-2)^ky^{\alpha k}}(-1)^k \sin (-k\pi \alpha) \quad \text{Equation 7}$$
+
+Note that $0 < \alpha < 1$ for $p > 2$. The $V_k$ terms are both positive and negative. This will limit the numerical accuracy that is obtained in summing it as shown in the following image. Again, if weights are applied to each row of the dataset, $equation 6$ becomes:
```

Comment:
The `$equation 6$` gets rendered like $equation 6$ which looks weird to me. We can either emphasize the equation  (by default italic in latex) like so `\emph{equation~6}` the `~` makes a non-breakable space so that we won't get "equation" on one line and "6" on the next one. Or we can use normal text like `equation~6`.


But in general, I'd suggest to use something like `equation~\ref{eq:6}` and modify the  equation 6 to something like:
```latex
\begin{equation}\label{eq:6}
$a(y, \phi, p) = \frac{1}{\pi y}V(y,\phi, p)
\end
```

With proper labels it will make it much less error prone when moving sections around as the latex will number the equations by itself.

Some introduction to cross-referencing: https://www.overleaf.com/learn/latex/Cross_referencing_sections%2C_equations_and_floats
More in detail: https://en.wikibooks.org/wiki/LaTeX/Labels_and_Cross-referencing

Reply:
Oh, love this -- thank you for directing me to cross-referencing!